src/Microsoft.Graph.Core/Exceptions/AuthenticationException.cs

```diff
@@ -0,0 +1,29 @@
+﻿// ------------------------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+// ------------------------------------------------------------------------------
+
+namespace Microsoft.Graph
+{
+    using System;
+    /// <summary>
+    /// Graph authentication exception
```

Comment:
This isn't really Graph authentication exception. This is an unknown authentication exception.. when MSAL throws an unknown exception, as in not an MSALException.